### ISSUE TITLE
dein yaml config.

### ISSUE THREAD

**Issue by taigacute** [Genki-S/dotfiles#2]
i  check the https://github.com/rafi/vim-config/blob/master/config/plugins.yaml, he doest use `lazy:1`, your config use lazy:1  whats different？

**Comment by Genki-S** [comment]
I assume you are asking the difference between the following 2:
```
repo/plugin:
  - lazy: 1
  - on_ft: go
```
```
repo/plugin:
  - on_ft: go
```

I specify `lazy: 1` because I did not know setting `on_*` option automatically marks a plugin lazy (today I learned dein does this [here](https://github.com/Shougo/dein.vim/blob/6040e7efeb824058f044b2175cc635817e66fd36/autoload/dein/parse.vim#L105-L117)).
It seems `lazy: 1` is redundant, and you don't have to specify it if you specify `on_*` option.

That being said I like explicitness, so I think I'd just keep `lazy: 1` for my config. It's just a matter of your preference, I think.

Hope this helps :)
